### ISSUE TITLE
Test images for processing

### ISSUE THREAD

**Issue by npilshchikova** [parseq/digital-phenotyping#1]
Для начала работы необходимы тестовые изображения.

Необходимо:

* сделать несколько фотографий (3 - 4) в заданных условиях (посмотреть, что предлагают в соответствующих статьях)
* посмотреть, есть ли подходящие фотографии в публикациях

Предлагаю начать с цвета кожи, т.к. этот признак кажется наиболее простым с точки зрения анализа изображений. 
Публикации по теме:

* Comprehensive candidate gene study highlights UGT1A and BNC2 as new genes determining continuous skin color variation in Europeans
* Genome-Wide Association Studies of Quantitatively Measured Skin, Hair, and Eye Pigmentation in Four European Populations

**Comment by miryad** [comment]
Примеры изображений есть в статье Fan Liu, но там у них очень низкое разрешение. 

Основная идея -- сфотографировать несколько образцов кожи при одинаковом освещении. 
Какой именно это участок не так существенно, для тестирования алгоритма по распознаванию цветового значения (hue/saturation) это могут быть даже разные участки кожи одного тела (разные по степени загара / пигментации).

**Comment by npilshchikova** [comment]
Да, основная идея именно такая.
Согласна с тем, что лучше взять несколько разных вариантов по степени загара (например, два), чтобы посмотреть, будут ли они различаться по hue/saturation, и параллельно - несколько одинаковых, но независимо сфотографированных, чтобы посмотреть, будут ли они совпадать.

По изображениям из публикаций: в статьях по предсказанию пигментации подходящих фотографий я не видела (везде очень низкое разрешение), возможно, стоит поискать в публикациях по дерматологии, но не факт, что там что-то будет. Думаю, сейчас это не первоочередная задача.

PS Добавила фотографию кожи с сайта Snipper 2.0 - самое подходящее из того, что я видела, но там всё равно сравнительно низкое разрешение.

**Comment by npilshchikova** [comment]
По фотографированию: можно поискать полиграфические калибровочные таблицы (в идеале, для HSV) и попробовать их использовать в качестве внутреннего стандарта.
Ещё одна идея - выполнить внутренний стандарт в форме рамки подходящего размера, по краям которой будет расположена калибровочная таблица. Если рамка будет иметь чёткий край, это позволит однозначно выделять ROI в пределах кадра.

**Comment by npilshchikova** [comment]
Ещё можно посмотреть на базу Humanae (http://humanae.tumblr.com/) - интересный проект, возможно, удастся его как-нибудь использовать.

**Comment by miryad** [comment]
Спасибо за ссылку. Там, по сути воплощена идея? За исключением того, что фоновый цвет, похоже, подбирался вручную.

**Comment by npilshchikova** [comment]
Нет, идея этого проекта принципиально иная, но воплощение очень интересное) 
Фоновый цвет, если верить описанию, задан на основе "усреднённого" цвета 11 пикселей с носа каждой из моделей, для него подобран соответствующий цвет из палитры PANTONE (идентификатор на каждой фотографии).

**Comment by npilshchikova** [comment]
Можно попробовать использовать фотографии из проекта Humanae для того, чтобы очертить диапазон возможных изменений цветов кожи/глаз/волос (если там достаточное разрешение для выделения цвета глаз) - они сделаны профессиональным фотографом при контролируемом освещении...

**Comment by miryad** [comment]
к текущему моменту:

1)принципиальная схема работы с кругами, по-моему, выглядит так:
1.1 открыть изображение, измерить, получить область его минимальных и максимальных цветовых значений, получить диапазон этих значений;
1.2 определить самую светлую облась 
1.3 вырезать ее.

2) из этого я пока что могу только загрузить, измерить, и получить координаты самой светлой / самой темной точки.

3) работа с Humanae -- это как внутренний стандарт -- то есть "какие в принципе могут быть значения?"

**Comment by miryad** [comment]
Единственное, что, похоже, подходит для работы с Humanae -- это Open.CV, там можно лица распознавать, а на лицах -- глаза
